src/sidebar/components/action-button.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+const classnames = require('classnames');
+const propTypes = require('prop-types');
+const { createElement } = require('preact');
+
+const SvgIcon = require('./svg-icon');
+
+/**
+ * A button, with (required) text label and an (optional) icon
+ */
+function ActionButton({
```

Comment:
I'm not sure about the name here, since all buttons trigger some kind of action when clicked. The distinctive features of this button compared to `IconButton` is that it has a background and a mandatory label. `TextButton`?

I wonder if perhaps it would make sense to combine this component and `IconButton` in future into one (`Button`?) with a prop that selects from one of the three styles (icon only, primary, secondary).